scanner/enricher/fixedby/fixedby.go

```diff
@@ -28,13 +28,11 @@ import (
 	"github.com/quay/claircore/suse"
 	"github.com/quay/claircore/ubuntu"
 	"github.com/quay/zlog"
+	"github.com/stackrox/rox/pkg/scannerv4/enricher/fixedby"
 )
 
 const (
-	// Type is the type of data returned from the Enricher's Enrich method.
-	Type = "message/vnd.stackrox.scannerv4.fixedby; enricher=fixedby"
-
-	// This appears above and must be the same.
+	// This appears in [fixedby.Type] and must be the same.
```

Comment:
```suggestion
	// This must be the same as the one in [fixedby.Type].
```
For an outsider, "fixedby" and "message/vnd.stackrox.scannerv4.fixedby; enricher=fixedby" are not quite the same. So who to trust here : )?

Reply:
I've tethered the name to the type in [04591be](https://github.com/stackrox/stackrox/pull/12310/commits/04591be133538e3ee3bb71d304d15619d495de5b)